Bottom: a7860f04a856dd41d346e747727bd2dfd6a0017d
Top:    3188021b4af79c625ac9988e41d094ef2843138d
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-26 18:23:45 +0800

Refresh of debug

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 83eefbc..8cfde21 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2291,6 +2291,7 @@ int vhost_get_vq_desc(struct vhost_virtqueue *vq,
 			}
 			*out_num += ret;
 		}
+		printk("vq %p head %d desc.addr %p\n", vq, head, desc->addr);
 		if (next_desc(vq, desc) == -1)
 			break;
 		desc = vhost_next_desc(vq, &ring_head, false);
